Gemfile.base

```diff
@@ -167,8 +167,8 @@ group :development do
 
   # Rails 4.2 have it built in
   gem 'quiet_assets'
-
-  gem 'yard', require: false
+ 
+  gem 'yard', '~> 0.9.11', require: false
```

Comment:
Why do we need to pin to a patch version? `~> 0.9` should be enough no? Then `bundle update` would keep working as expected.